graylog2-web-interface/src/views/logic/searchtypes/aggregation/PivotConfigGenerator.js

```diff
@@ -84,7 +84,7 @@ export default ({ config }: { config: AggregationWidgetConfig }) => {
 };
 
 class ConfigBuilder {
-  value: Set;
+  value: Set<any>;
```

Comment:
Instead of any, we can use an object here with `{  id: string, name: string, type: string, config?: {}}`. I could not find a different use case.

Reply:
Added that, thanks!